### PR TITLE
[mapnik] test static build

### PR DESCRIPTION
**Pull request by mathisloge**
**Describe the pull request**

- #### What does your PR fix?  
enables mapnik static linkage.

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
hopefully all except supported

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
I am still working on this PR

### CODE REVIEW

File: ports/mapnik/portfile.cmake [hunk 1]
```diff
@@ -1,11 +1,9 @@
 # test application for this port: https://github.com/mathisloge/mapnik-vcpkg-test
-
-vcpkg_check_linkage(ONLY_DYNAMIC_LIBRARY)
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
-    REPO mapnik/mapnik
-    REF d7b83c0f7d11397aff5b5d8e0bb294ef6ea4354d
-    SHA512 62b47cb753e9698e55fe88593009016676b6c0c0a90c3f29be6f44a45f9f783ec5beca6916e549f9adbdc750e2e0334a9e927fc0dcb6a88431e40c2e920ff962
+    REPO mathisloge/mapnik
```
Comment (LilyWangLL) [inline ports/mapnik/portfile.cmake:4]: Why change to a unofficial github url?

**Review by LilyWangLL — CHANGES_REQUESTED** [review]

File: ports/mapnik/portfile.cmake (hunk 1 above)
Comment (mathisloge) [reply ports/mapnik/portfile.cmake:4]: I'm currently testing static linkage. The static build is in a PR for upstream.

This PR is a draft as long as the upstream PR is not merged. I'll then change back to the official repo

**Comment by LilyWangLL** [comment]
Ping @mathisloge for response. Does the upstream PR has new updates?

**Comment by mathisloge** [comment]
@LilyWangLL i will update the PR if I have resolved all bugs at upstream.

**Comment by LilyWangLL** [comment]
Closing this PR since it seems that no progress is being made. Please reopen if work is still being done.

**Pull request closed without merging**